src/actions/GraphDataActions.ts

```diff
@@ -111,26 +135,10 @@ export const GraphDataActions = {
             dispatch(MessageCenterActions.addMessage(emsg));
             dispatch(GraphDataActions.getGraphDataFailure(emsg));
           }
-        );
-      }
-      return API.getGraphElements(authentication(), namespace, restParams).then(
-        response => {
-          const responseData: any = response['data'];
-          const graphData = responseData && responseData.elements ? responseData.elements : EMPTY_GRAPH_DATA;
-          const timestamp = responseData && responseData.timestamp ? responseData.timestamp : 0;
-          dispatch(GraphDataActions.getGraphDataSuccess(timestamp, graphData));
-        },
-        error => {
-          let emsg: string;
-          if (error.response && error.response.data && error.response.data.error) {
-            emsg = 'Cannot load the graph: ' + error.response.data.error;
-          } else {
-            emsg = 'Cannot load the graph: ' + error.toString();
-          }
-          dispatch(MessageCenterActions.addMessage(emsg));
-          dispatch(GraphDataActions.getGraphDataFailure(emsg));
-        }
-      );
+        )
+        .catch(err => {
+          console.debug(API.getErrorMsg('Cannot load the graph: ', err));
+        });
```

Comment:
Is the goal to log messages to the console?
The `then` block is already "catching" errors because two arguments are being passed. (See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise#Methods_2)

You could log to the console in the second argument of the `then` (the `error` arrow fn).

TBH, I prefer to have individual `then` and `catch` blocks because it makes more evident what code is handling the success and what code is catching errors. So, if you want to "split" the `then` I will be happy to merge that change.

Reply:
@israel-hdez No, it's fine.  This is why I asked for review.  I understand now that this would be redundant handling. Thanks, I'm going to close this.